### PR TITLE
Create services/repos/controllers

### PR DESCRIPTION
**Pull request by vjwua**
The name is self-descriptive

### CODE REVIEW

File: src/main/java/com/pnudev/springonlinedelivery/controllers/MenuItemController.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping(path = "/menu-items")
+public class MenuItemController {
+
+    private MenuItemService menuItemService;
```
Comment (justme1135) [inline src/main/java/com/pnudev/springonlinedelivery/controllers/MenuItemController.java]: make it `final`

File: src/main/java/com/pnudev/springonlinedelivery/controllers/OrderController.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import com.pnudev.springonlinedelivery.models.Order;
+
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping(path = "/orders")
+public class OrderController {
+
+    private OrderService orderService;
```
Comment (justme1135) [inline src/main/java/com/pnudev/springonlinedelivery/controllers/OrderController.java]: same here

File: src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping(path = "/users")
+public class UserController {
+
+    private UserService userService;
+    @GetMapping
```
Comment (getmane) [inline src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java:18]: Whitespace before annotation (related to all such issues)

File: src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import com.pnudev.springonlinedelivery.models.User;
+
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping(path = "/users")
+public class UserController {
+
+    private UserService userService;
```
Comment (justme1135) [inline src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java]: same here

File: src/main/java/com/pnudev/springonlinedelivery/service/OrderServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService{
+    @Override
+    public List<Order> getOrders() {
+        return Collections.emptyList();
```
Comment (justme1135) [inline src/main/java/com/pnudev/springonlinedelivery/service/OrderServiceImpl.java:14]: Create a //TODO for unimplemented features

File: src/main/java/com/pnudev/springonlinedelivery/service/UserServiceImpl.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Override
+    public List<User> getUsers() {
+        return Collections.emptyList();
```
Comment (justme1135) [inline src/main/java/com/pnudev/springonlinedelivery/service/UserServiceImpl.java:14]: Same here

**Review by justme1135 — CHANGES_REQUESTED** [review]

File: src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java (hunk 3 above)
Comment (vjwua) [reply src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java:18]: Fixed

File: src/main/java/com/pnudev/springonlinedelivery/controllers/MenuItemController.java (hunk 1 above)
Comment (vjwua) [reply src/main/java/com/pnudev/springonlinedelivery/controllers/MenuItemController.java]: Fixed

File: src/main/java/com/pnudev/springonlinedelivery/controllers/OrderController.java (hunk 2 above)
Comment (vjwua) [reply src/main/java/com/pnudev/springonlinedelivery/controllers/OrderController.java]: Fixed

File: src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java (hunk 4 above)
Comment (vjwua) [reply src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java]: And here fixed

File: src/main/java/com/pnudev/springonlinedelivery/service/UserServiceImpl.java (hunk 6 above)
Comment (vjwua) [reply src/main/java/com/pnudev/springonlinedelivery/service/UserServiceImpl.java:14]: Done

File: src/main/java/com/pnudev/springonlinedelivery/service/OrderServiceImpl.java (hunk 5 above)
Comment (vjwua) [reply src/main/java/com/pnudev/springonlinedelivery/service/OrderServiceImpl.java:14]: Done

File: src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java (hunk 3 above)
Comment (vjwua) [reply src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java:18]: So there should be tab?

Comment (getmane) [reply src/main/java/com/pnudev/springonlinedelivery/controllers/UserController.java:18]: Yes

**Review by getmane — APPROVED** [review]

**Review by getmane — APPROVED** [review]

**Review by justme1135 — APPROVED** [review]

**Pull request merged**